tools/cooler/cooler_zoomify.xml

```diff
@@ -0,0 +1,47 @@
+<tool id="cooler_zoomify" name="cooler_zoomify" version="@VERSION@+galaxy0" profile="18.01">
+  <description>Copy and balance a cool file.</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements" />
+  <expand macro="stdio" />
+  <command detect_errors="exit_code"><![CDATA[
+    cooler zoomify -p \${GALAXY_SLOTS:-4}
+    -c $chunksize
+    -r '$resolution'
```

Comment:
```suggestion
    -r '$r'
```

Reply:
You only specified the short argument form - and did not overwrite it by using `name="resolution"`.